onnxruntime/test/providers/cpu/rnn/deep_cpu_gru_op_test.cc

```diff
@@ -96,7 +96,9 @@ static void RunGruTest(const std::vector<float>& X_data,
   } else {
     test.AddMissingOptionalOutput<float>();
   }
-  test.Run();
+  
+  // TensorRT failed on GRU tests
```

Comment:
can this be investigated further? 
TRT7 is supposed to improve RNN support.

Reply:
Both RNN failures are under investigation.